config/manifests/bases/kogito-serverless-operator.clusterserviceversion.yaml

```
@@ -15,7 +15,7 @@ spec:
       displayName: Kogito Serverless Workflow
       kind: KogitoServerlessWorkflow
       name: kogitoserverlessworkflows.kie.kogito.sw.org
```

```suggestion
      name: kogitoserverlessworkflows.sw.kogito.kie.org
```

I think we need to review the naming.